<ui version="4.0" >
 <class>qdesigner_internal::NewDynamicPropertyDialog</class>
 <widget class="QDialog" name="qdesigner_internal::NewDynamicPropertyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create Dynamic Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="1" >
      <widget class="QLineEdit" name="m_lineEdit" >
       <property name="minimumSize" >
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Property Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QComboBox" name="m_comboBox" />
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="spacerName" stdset="0" >
          <string>horizontalSpacer</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Property Type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="spacerName" stdset="0" >
      <string/>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
